ambry-frontend/src/main/java/com/github/ambry/frontend/FrontendRestRequestServiceFactory.java

```diff
@@ -89,9 +92,12 @@ public RestRequestService getRestRequestService() {
               clusterMap.getMetricRegistry()).getUrlSigningService();
       AccountAndContainerInjector accountAndContainerInjector =
           new AccountAndContainerInjector(accountService, frontendMetrics, frontendConfig);
+      QuotaManagerFactory quotaManagerFactory =
+          new AmbryQuotaManagerFactory(new QuotaConfig(verifiableProperties), Collections.emptyList(),
+              Collections.emptyList());
```

Comment:
In RestServerMain we already created a quotaManagerFactory, why we are creating one again here?